networkx/generators/community.py

```diff
@@ -285,7 +285,7 @@ def planted_partition_graph(l, k, p_in, p_out, seed=None, directed=False):
     Raises
     ------
     NetworkXError
-      If p_in,p_out are not in [0,1] or
+      If p_in,p_out are not in [0,1]
```

Comment:
```suggestion
      If `p_in`, `p_out` are not in `[0, 1]`
```

might as well make it a bit more pretty :)